mmcv/ops/saconv.py

```diff
@@ -98,6 +98,8 @@ def forward(self, x):
         switch = self.switch(avg_x)
         # sac
         weight = self._get_weight(self.weight)
+        zero_bias = torch.zeros(self.out_channels, device=weight.device)
```

Comment:
bias should have similar dtype also as weight. Otherwise it will fail in FP16 mode.